src/SFML/Graphics/Font.cpp

```diff
@@ -41,6 +41,7 @@
 #include FT_OUTLINE_H
 #include FT_BITMAP_H
 #include FT_STROKER_H
+#include <cassert>
```

Comment:
Is this a leftover from testing or sometimes I'm missing? I don't see any new asserts and it compiles fine for me without.

Reply:
It is -- will remove it.